test/test_text_check.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: false
+
+module REXMLTests
+  class TextCheckTester < Test::Unit::TestCase
+    include REXML
+
+    def test_valid_pattern
+      chars = ['&A;', '&:;', '&_;', '&:A;', '&AA.bb-00;']
```

Comment:
Do we need both of `&:;` and `&_;`?
Do we need `&:A;`?

Reply:
> Do we need both of &:; and &_;?
The characters that can be used individually are specified in:
https://github.com/ruby/rexml/blob/b2ec329dc1dc7635b224a6d61687c24b1e1db6fd/lib/rexml/xmltokens.rb#L26-L29

> Do we need &:A;?
Hmm, we can remove it...